src/components/BalanceGame/index.tsx

```diff
@@ -0,0 +1,76 @@
+import { useState } from "react";
+import { BALANCEGAME } from "./data";
+import Header from "./Header";
+import * as S from "./styled";
+import { useNavigate } from "react-router-dom";
+
+const BalanceGame = () => {
+  const [num, setNum] = useState(0);
+  const maxNum = 5;
+  const progressPercentage = Math.floor(((num + 1) / maxNum) * 100);
+  const navigate = useNavigate();
+
+  const handleNextQuestion = () => {
+    if (num + 1 < maxNum) {
+      setNum(num + 1);
```

Comment:
```suggestion
      setNum((prev) => prev + 1);
```
함수형 업데이트를 사용해봐도 좋을 것 같아요~